src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/ProjectSystem/GeneratedDocumentContainerTest.cs

```diff
@@ -101,12 +104,20 @@ public void SetOutput_InvokesChangedEvent()
             container.GeneratedHtmlChanged += (o, a) => { htmlChanged = true; };
 
             // Act
-            container.SetOutput(document, csharpDocument, htmlDocument, version, version, version);
+            container.SetOutput(document, codeDocument, version, version, version);
 
             // Assert
             Assert.NotNull(container.LatestDocument);
             Assert.True(csharpChanged);
             Assert.True(htmlChanged);
         }
+
+        private static RazorCodeDocument CreateCodeDocument(RazorCSharpDocument csharpDocument, RazorHtmlDocument htmlDocument)
+        {
+            var codeDocument = TestRazorCodeDocument.CreateEmpty();
+            codeDocument.SetCSharpDocument(csharpDocument);
+            codeDocument.Items[typeof(RazorHtmlDocument)] = htmlDocument;
```

Comment:
Out of curiosity, is there a reason why the C# doc has its own dedicated Set method, but the HTML doc doesn't?

Reply:
Great question, I thought the same thing too but for some reason I guess we haven't needed it?